mcs/class/System.Drawing/testing_aot_full_System.Drawing_xtest.dll.exclude.sources

```diff
@@ -0,0 +1,5 @@
+../../../external/corefx/src/System.Drawing.Common/tests/Helpers.cs
```

Comment:
We should do include only relevant tests instead

Reply:
I'm just basing this off of 75f3c0cc0c8d0b7e7aa2463e750dbbd2a83420ef. 
I can copy and paste the netstandard template instead if you prefer. I was trying to keep the style consistent.